src/events/AdminEvents.tsx

```diff
@@ -562,7 +569,13 @@ export const AdminEvents = () => {
           data-testid="representation-dialog"
           onClose={() => setRepresentationEvent(undefined)}
         >
-          some json from the changed values
+          <CodeEditor
+            isLineNumbersVisible={true}
+            isReadOnly={true}
```

Comment:
Boolean props that are true can also be provided without assigning a value.
```suggestion
            isLineNumbersVisible
            isReadOnly
```

Reply:
thanks! I overlooked this